### PR TITLE
added the navbar component

### PR DESCRIPTION
**Pull request by Fatemeh-Pakpour**

### CODE REVIEW

File: src/client/components/NavBar/NavBar.component.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./NavBar.css";
+
+const NavBar = ({ title, navElements }) => {
+  const navList = navElements.map(item => {
+    return (
+      <a href="#">
```
Comment (bytewiz) [inline src/client/components/NavBar/NavBar.component.js]: Maybe there is should a prop for the `href` as well so you can pass in a link? and then the default can be `"#"`? :)

File: src/client/components/NavBar/NavBar.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
+@import url("https://fonts.googleapis.com/css?family=Open+Sans|Pacifico&display=swap");
```
Comment (bytewiz) [inline src/client/components/NavBar/NavBar.css]: This should go in a general `css` file and not for a specific component 👍 so we can remove it from here

File: src/client/components/NavBar/NavBar.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+.nav-elements li {
+   padding: 5px 30px ;  
+}
+.nav-bar {
+  display: flex;
+  flex-direction: row;
+  float:right;
+  margin:30px 10px 0 10px;
+}
+a:link {
```
Comment (bytewiz) [inline src/client/components/NavBar/NavBar.css]: be aware of using the `global` selectors unless you have 100% sure all `<a>` should look the exact same and if so this should probably go in a generic `css` file that is loaded for the entire app and not only for the component

File: src/client/components/NavBar/NavBar.story.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import NavBar from "../NavBar/NavBar.component";
+
+storiesOf("NavBar", module).add("with title ", () => (
+  <NavBar
+    title="Register"
+    navElements={[
+      { title: "Annotate", id: 1 },
```
Comment (bytewiz) [inline src/client/components/NavBar/NavBar.story.js:8]: if we added prop for links then we can add here as well 👍

File: src/client/stories/index.js [hunk 5]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
 import { Button } from "@storybook/react/demo";
+import {NavBar} from "../components/NavBar/NavBar.story";
 
 storiesOf("Button", module)
```
Comment (bytewiz) [inline src/client/stories/index.js]: Can we delete this one? 👍 it is no longer used

File: src/client/stories/index.js [hunk 6]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import { storiesOf } from "@storybook/react";
 import { Button } from "@storybook/react/demo";
```
Comment (bytewiz) [inline src/client/stories/index.js]: Can we delete this one? 👍 it is no longer used

**Review by bytewiz — CHANGES_REQUESTED** [review]
Looks good only a few minor comments 👍

File: src/client/components/NavBar/NavBar.story.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+      { title: "About", id: 2 },
+      { title: "Login", id: 3 }
+    ]}
+    href ="#"
```
Comment (bytewiz) [inline src/client/components/NavBar/NavBar.story.js:12]: I think I would add the `href`prop for each of the elements so they can each have a different link

Comment (bytewiz) [reply src/client/components/NavBar/NavBar.story.js:12]: @Fatemeh-Pakpour ☝️

**Review by bytewiz — CHANGES_REQUESTED** [review]

File: src/client/components/RegisterButton/RegisterButton.css [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+    border-radius: 40px;
+    background-color:white;
+    border: 1px solid black;
+    width: 215px;
+    height: 80px;
+    color:#2D2E36;  
+    font-family: "Open Sans";
+    font-size: 25px;
+    font-weight: bold;
+    letter-spacing: 1px;
```
Comment (laila1222) [inline src/client/components/RegisterButton/RegisterButton.css:11]: Maybe add cursor: pointer here? 
So it also looks like it's a button when we hover over it.

**Review by laila1222 — COMMENTED** [review]
I just had one suggestion in the register button css file. 👍 
The tests didn't run for me ('No tests found related to files changed since last commit.'), but it might have been something to do with me, as earlier today it worked.

**Comment by Laila1222** [comment]
> I just had one suggestion in the register button css file. 👍
> The tests didn't run for me ('No tests found related to files changed since last commit.'), but it might have been something to do with me, as earlier today it worked.

This test problem I described shows error only on my computer locally, as it worked for Olha. So nothing to do for you related to it ;)

File: src/client/components/RegisterButton/RegisterButton.css (hunk 8 above)
Comment (OlkaA) [reply src/client/components/RegisterButton/RegisterButton.css:11]: Great job, so much work done.
My only suggestion is to add styling for mobile version, so items could be shown in column.

File: src/client/components/NavBar/NavBar.component.js [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+import "./NavBar.css";
+
+const NavBar = ({navElements, href }) => {
+  const navList = navElements.map(item => {
+    return (
+      <a className="nav-link" href={href} key={item.id}>
+        <li className="nav-style" key={item.id}>
+          {item.title}
+        </li>
+      </a>
```
Comment (OlkaA) [inline src/client/components/NavBar/NavBar.component.js]: It renders, but isn't "a" should be inside "li" element?
Like this:
ul -> li -> a -> Text -< /a -< /li -< /ul

File: src/client/components/NavBar/NavBar.test.js [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+        { title: "Annotate", id: 1 },
+        { title: "About", id: 2 },
+        { title: "Login", id: 3}
+      ];
+    const href = "#";
+    const {container} = render(<NavBar 
+    navElements ={navElements}
+    href ={href}
+    />)
+    expect(container.children.length).toBe(1);
```
Comment (OlkaA) [inline src/client/components/NavBar/NavBar.test.js]: I am not quite sure how this test is working.
I have a suggestion, maybe it is better to do it something like this:
expect(container.getElementsByTagName('a')[0].innerHTML).toBe('Annotate');

**Review by OlkaA — CHANGES_REQUESTED** [review]

File: src/client/components/NavBar/NavBar.test.js (hunk 10 above)
Comment (Fatemeh-Pakpour) [reply src/client/components/NavBar/NavBar.test.js]: Thank you for your feedback, I used your suggestion and that worded.

**Review by OlkaA — APPROVED** [review]

File: src/client/components/NavBar/NavBar.component.js (hunk 1 above)
Comment (Fatemeh-Pakpour) [reply src/client/components/NavBar/NavBar.component.js]: I fixed that.

File: src/client/components/NavBar/NavBar.story.js (hunk 7 above)
Comment (Fatemeh-Pakpour) [reply src/client/components/NavBar/NavBar.story.js:12]: I fixed that.

File: src/client/components/RegisterButton/RegisterButton.css (hunk 8 above)
Comment (Fatemeh-Pakpour) [reply src/client/components/RegisterButton/RegisterButton.css:11]: I added cursor:pointer  to the register button and the nav links.

File: src/client/components/NavBar/NavBar.component.js (hunk 9 above)
Comment (Fatemeh-Pakpour) [reply src/client/components/NavBar/NavBar.component.js]: I did it.

**Comment by Fatemeh-Pakpour** [comment]
@bytewiz could you please approve my pull request?

**Review by bytewiz — APPROVED** [review]

**Pull request merged**